wikibase.api.js

```diff
@@ -28,51 +71,39 @@ class WikibaseApi {
 
 		Object.assign( itemData, { labels }, data );
 
-		const bot = new MWBot( {
-			apiUrl: `${browser.config.baseUrl}/api.php`
-		} );
+		const bot = this.getBot();
 
-		return bot.getEditToken()
-			.then( () => {
-				return bot.request( {
-					action: 'wbeditentity',
-					new: 'item',
-					data: JSON.stringify( itemData ),
-					token: bot.editToken
-				} );
-			} ).then( ( response ) => {
-				return response.entity.id;
-			} );
+		return bot.request( {
```

Comment:
Won’t this fail because `bot` is a Promise instead of an MWBot? We’d need something like this, I think:

```js
return this.getBot()
    .then( ( bot ) => bot.request( {
        action: 'wbeditentity',
```

Reply:
True. OTOH, I wonder if we can use `async`/`await` by now and just have `const bot = await this.getBot();`?

Reply:
Yeah, it looks like async/await is available in Node 12 (or even Node 10 already). Might need a change to some eslint configs, but apart form that it should be fine, I think.